.github/workflows/ci_cd.yml

```diff
@@ -41,8 +41,8 @@ jobs:
         uses: actions/setup-python@v4
         with:
           python-version: ${{ matrix.python-version }}
-          cache: 'pip'
-          cache-dependency-path: requirements_test.txt
+          # cache: 'pip'
+          # cache-dependency-path: requirements_test.txt
```

Comment:
```suggestion
          cache: 'pip'
          cache-dependency-path: pyproject.toml
```

Reply:
The way ``cache-dependency-path`` works is that it looks for the hash of a certain file (i.e. it's checksum). If the checksum is modified (i.e. your file has changed) it will update the cache =)